agenta-web/src/components/EvaluationTable/SimilarityMatchEvaluationTable.tsx

```diff
@@ -0,0 +1,388 @@
+import {useState, useEffect} from "react"
+import type {ColumnType} from "antd/es/table"
+import {LineChartOutlined} from "@ant-design/icons"
+import {Button, Card, Col, Input, Row, Space, Spin, Statistic, Table, Tag} from "antd"
+import {Variant} from "@/lib/Types"
+import {updateEvaluationRow, callVariant} from "@/lib/services/api"
+import {useVariant} from "@/lib/hooks/useVariant"
+import {useRouter} from "next/router"
+import {EvaluationFlow} from "@/lib/enums"
+import {evaluateWithSimilarityMatch} from "@/lib/services/evaluations"
+
+interface SimilarityMatchEvaluationTableProps {
+    appEvaluation: any
+    columnsCount: number
+    evaluationRows: SimilarityMatchEvaluationTableRow[]
+}
+
+interface SimilarityMatchEvaluationTableRow {
+    id?: string
+    inputs: {
+        input_name: string
+        input_value: string
+    }[]
+    outputs: {
+        variant_name: string
+        variant_output: string
+    }[]
+    columnData0: string
+    correctAnswer: string
+    score: string
+    evaluationFlow: EvaluationFlow
+}
+/**
+ *
+ * @param appEvaluation - Evaluation object
+ * @param evaluationRows - Evaluation rows
+ * @param columnsCount - Number of variants to compare face to face (per default 2)
+ * @returns
+ */
+
+const SimilarityMatchEvaluationTable: React.FC<SimilarityMatchEvaluationTableProps> = ({
+    appEvaluation,
+    evaluationRows,
+    columnsCount,
+}) => {
+    const router = useRouter()
+    let app_name = ""
+    if (Array.isArray(router.query.app_name)) {
+        app_name = router.query.app_name[0]
+    } else if (typeof router.query.app_name === "string") {
+        app_name = router.query.app_name
+    }
```

Comment:
**Suggestion:** Avoid using `let` for variable declaration when the variable is not reassigned. Use `const` instead for better readability and to prevent unintentional reassignment.
```suggestion
    const app_name = Array.isArray(router.query.app_name) ? router.query.app_name[0] : router.query.app_name || "";
```